app/src/pages/index.tsx

```diff
@@ -29,7 +28,7 @@ const Index: NextPage<{ marketId: AccountId }> = ({ marketId }) => {
 export const getServerSideProps = async ({ locale }: GetServerSidePropsContext) => {
   await i18n?.reloadResources();
 
-  const marketId = await pulse.promptWars.getLatestMarketId();
+  const marketId = "0xE82d465514305DF28E6C86E3ab429c579DC8499a";
```

Comment:
@Udit-takkar this is our deployed contract address in the [OP Sepolia chain](https://optimism-sepolia.blockscout.com/address/0xE82d465514305DF28E6C86E3ab429c579DC8499a?tab=txs).

You can add this chain by following these steps: https://chainlist.org/chain/11155420